.github/workflows/rav1e.yml

```diff
@@ -280,9 +278,14 @@ jobs:
         $NASM_VERSION="nasm-2.14.02"
         curl -LO "https://people.xiph.org/~tdaede/$NASM_VERSION-win64.zip"
         7z e -y "$NASM_VERSION-win64.zip" -o"C:\nasm"
-    - name: Set environment variables
+        echo "::add-path::C:\nasm"
+    - name: Set MSVC x86_64 linker path
+      shell: pwsh
```

Comment:
Is it possible to use the `Desktop Poweshell` (default shell used on Windows for GA) instead of the `PowerShell Core`?

Reply:
Yes, from the logs we see that the exact same shell binary is run without explicitly setting `shell: pwsh`.

Reply:
Great! :)